io.itch.domestique_baston.domestique_baston.yml

```diff
@@ -0,0 +1,32 @@
+app-id: io.itch.domestique_baston.domestique_baston
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: domestique_baston
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+  - --socket=pulseaudio
+modules:
+  - name: domestique_baston
+    buildsystem: simple
+    build-commands:
+      - install -D DOMESTIQUE_BASTON_gles3.x86_64 /app/bin/domestique_baston
+      - install -D -m 0644 *.pdf /app/bin
+      - install -D -m 0644 io.itch.domestique_baston.domestique_baston.metainfo.xml /app/share/metainfo/io.itch.domestique_baston.domestique_baston.metainfo.xml
+      - install -D -m 0644 io.itch.domestique_baston.domestique_baston.desktop /app/share/applications/io.itch.domestique_baston.domestique_baston.desktop
+      - install -D -m 0644 icon96.png /app/share/icons/hicolor/96x96/apps/io.itch.domestique_baston.domestique_baston.png
+      - install -D -m 0644 icon48.png /app/share/icons/hicolor/48x48/apps/io.itch.domestique_baston.domestique_baston.png
+    sources:
+      - type: archive
+        url: 'http://www.ferdiferdi.com/db/DOMESTIQUE_BASTON_LINUX.zip'
```

Comment:
the URL is not versioned which mean that when there is a new release, things will break. It's a bad practice.